src/Theme_Language_Command.php

```diff
@@ -114,8 +114,15 @@ function( $file ) {
 		$current_locale = get_locale();
 
 		$translations = array();
+		$themes      = new \WP_CLI\Fetchers\Theme();
```

Comment:
Alignment issue here:
```suggestion
		$translations = array();
		$themes       = new \WP_CLI\Fetchers\Theme();
```